descriptors/ModuleDescriptor-template.json

```diff
@@ -1739,8 +1757,8 @@
         "orders.rollover.item.post",
         "orders.holding-summary.collection.get",
         "orders.acquisition-methods.all",
-        "orders.export-history.all"
-
+        "orders.export-history.all",
+        "orders.routing-list-template.item.get"
```

Comment:
we should add this permission in routing-lists.all when this PR will be merged https://github.com/folio-org/mod-orders/pull/879